src/json.rs

```diff
@@ -15,6 +18,101 @@ pub struct LlvmCovJsonExport {
     pub(crate) version: String,
 }
 
+/// <https://docs.codecov.com/docs/codecov-custom-coverage-format>
+///
+/// The fraction
```

Comment:
This seems like a sentence in the process of writing.

Reply:
0175abe

Reply:
https://github.com/taiki-e/cargo-llvm-cov/blob/0175abeca07d49f820d0b2df50e927d3f0976372/src/json.rs#L21-L28